content/QRadar-SIEM/102.es.md

```diff
@@ -0,0 +1,1079 @@
+
+5.  Seleccione el cuadro de mandos **Descripción general del cumplimiento**.
+
+    **Sugerencia**: Debido a los eventos generados aleatoriamente, los datos del sistema en vivo pueden ser ligeramente diferentes de las capturas de pantalla de la guía de demostración.
+
+    ![](./images/102/image3.png)
+
+    Este panel centrado en el cumplimiento captura todos los casos de uso, también conocidos como reglas de QRadar, que se han activado en función de los eventos, el tráfico de red, las vulnerabilidades y la actividad de los usuarios en su red y activos relacionados con diferentes normativas de cumplimiento. El panel se suministra mediante el paquete de contenido del sitio IBM Security App Exchange.
+
+    > App Exchange es un portal en el que IBM y toda la comunidad de IBM Security pueden desarrollar funciones complementarias para QRadar SIEM core y otros productos de IBM Security. También puede crear sus propios paneles y portlets.
+
+    El cuadro de mandos de Cumplimiento tiene múltiples portlets o widgets. Muchos de estos gráficos son interactivos, y puede personalizar cada uno de estos widgets utilizando búsquedas de sentencias QRadar Ariel Query Language[(AQL](https://www.ibm.com/docs/SS42VS_7.4/com.ibm.qradar.doc/b_qradar_aql.pdf?view=kc)).
+
+6.  Céntrate en la esquina superior derecha del salpicadero.
+
+    Revise los _eventos de autenticación en el_ gráfico de barras _del servidor GDPR_ en la parte superior derecha.
+
+7.  Demuestre que los gráficos tienen algo de interactividad haciendo clic en las etiquetas **Inicio de sesión de usuario** e **Inicio de sesión fallido** para ocultar esos eventos y centrar la atención en el resto de los eventos generados.
+
+    ![](./images/102/image4.png)
+
+    Observe cómo el gráfico es interactivo. Al ocultar los datos atípicos, puedes centrar más fácilmente tu investigación en los datos importantes, como los distintos tipos de fallos de inicio de sesión en este caso.
+
+    > Opcionalmente, puede dedicar algún tiempo a revisar cada visualización, ya que proporciona una gran visión a un administrador o ejecutivo sobre todos sus servidores relevantes para el cumplimiento y los fallos de inicio de sesión observados en cada uno por nombre de usuario. Puede extraer los resultados de cualquier consulta en un gráfico o una tabla de visualización; estos son sólo algunos ejemplos.
+    >
+    > También tenga en cuenta que QRadar puede mostrar cuadros de mando y reposts basados en el MITRE ATT\&CK Framework. Más adelante hablaremos de ello con más detalle.
```

Comment:
I think `reposts` should be `reports` in the english version and `informa` in the Spanish version. @timroster please confirm if thats correct.

Reply:
@adams0619 yes `reposts` is a typo. agree with suggested changes will update English version along with URL updates.